src/cron/schedule-event-reminder.js

```diff
@@ -19,6 +19,24 @@ const clearAllCronJobs = () => {
   activeCronJobs = [];
 };
 
+const generateReminderCronExpression = (event) => {
+  const now = new Date();
+  const eventEndTimeDate = new Date(event.scheduledEndTimestamp);
+  const timeDifference = eventEndTimeDate - now;
+
+  const reminderTime = new Date(
+    timeDifference > dayInMilliseconds
+      ? eventEndTimeDate - dayInMilliseconds
+      : eventEndTimeDate - hoursInMilliseconds
+  );
+
+  if (reminderTime < now) {
+    return null;
+  }
```

Comment:
Maybe throwing an descriptive error is clearer, then you can catch it with a try catch in scheduleEventReminder